include/ddc/discrete_space.hpp

```diff
@@ -88,25 +88,25 @@ class gpu_proxy
     }
 
     DDC_INLINE_FUNCTION
-    T* data()
+    T* data_handle()
```

Comment:
Are we sure about that ?

Reply:
not at all :)

Reply:
I would do it to align with mdspan

Reply:
We need to understand why they switched to `data_handle()` because in the standard it is usually `data()` right ?

Reply:
This is explained here https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2022/p2604r0.html#pointer